src/boilerplates/boilerplates.ts

```diff
@@ -116,6 +117,21 @@ export function makeCommandBoilerplates(program: Command) {
             createEmbeddedWalletVerifySession(program, options);
         });
 
+    comm
+        .command("create-universal-wallet-react")
```

Comment:
If this is a boilerplate and also a starting point, I would call it create-universal-wallet-react-starter.

Reply:
Agreed – @moskalyk Also add the new boilerplate to the README please.